src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
```

Comment:
the `new` keyword in the variable name don't any meaning
```suggestion
  const inversedRobot = {};
```